lib/installedtest.pm

```diff
@@ -288,7 +288,12 @@ sub maybe_unlock_screen {
 sub save_and_upload_log {
     my ($self, $cmd, $file, $args) = @_;
```

Comment:
And now I wonder if this shouldn't be
```suggestion
    my ($self, $cmd, $file, %args) = @_;
```
That's what I see in testapi.pm for example.